test/runtests.jl

```diff
@@ -50,6 +50,8 @@ if GROUP == "All" || GROUP == "Interface"
     include("interface/ad.jl")
     include("interface/optimize.jl")
     include("interface/repgradelbo.jl")
+    include("interface/rules.jl")
+    include("interface/averaging.jl")
```

Comment:
Are these an unrelated fix, where the files were there but not being included? Just wondering since these files seem to not have been edited in this PR. I'm happy to have an unrelated fix in this PR, just want to understand.

Reply:
Not sure how this ended up here, but it seems like it went away now.